### PR TITLE
New field skill_level on events.

### PR DESCRIPTION
**Pull request by FloQuinn**
## Describe context / purpose for this PR
Add selector for Beginner/Intermediate/Advanced for edit events.  This is a new field. 
Also exposed this selector for upcoming events list selection.

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1678

## Any other related PRs?
no

## Link to MultiDev instance

http://md-1678-accessmatch.pantheonsite.io

## Checklist for PR author
- [x ] I have checked that the PR is ready to be merged
- [ x] I have reviewed the DIFF and checked that the changes are as expected
- [ x] I have assigned myself or someone else to review the PR

### CODE REVIEW

File: web/sites/default/config/default/views.view.project_submissions.yml
Comment (FloQuinn) [inline web/sites/default/config/default/views.view.project_submissions.yml:1]: Oops, made a mistake deleting this.  Trying to put back into this branch.

File: web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml [hunk 1]
```diff
@@ -112,6 +113,12 @@ content:
       placeholder_url: ''
       placeholder_title: ''
     third_party_settings: {  }
+  field_skill_level:
+    type: options_select
```
Comment (a-pasquale) [inline web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml:117]: We should make this a multiselect field to accommodate trainings that are suitable for multiple skill levels.

Comment (FloQuinn) [reply web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml:117]: I made the change locally, and it has a side effect that the event gets listed N times in the events lists,  where N is  the number of skill level items selected in all the events lists.  Do you want me to keep this change and try to figure out why that happens?

Comment (a-pasquale) [reply web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml:117]: There is a setting on the view to combine multiple instances. 
I think the multiple setting might make the filter not work anymore though.

Comment (FloQuinn) [reply web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml:117]: I changed the settings on the new filter. Seems to work, tested with various combos of filters

**Pull request merged**